lib/omnibus/packagers/bff.rb

```diff
@@ -208,7 +208,10 @@ def create_bff_file
       # This implies that if we are in /tmp/staging/project/dir/things,
       # we will chown from 'project' on, rather than 'project/dir', which leaves
       # project owned by the build user (which is incorrect)
-      shellout!("sudo chown -R 0:0 #{File.join(staging_dir, project.install_dir.match(/^\/?(\w+)/).to_s)}")
+      # First - let's find out who we are.
+      original_uid = shellout!("id -u").stdout.chomp
+      original_gid = shellout!("id -g").stdout.chomp
```

Comment:
Since these `original_*` vars are not used until the ensure block it's best to declare them down there.